StockTrader/src/main/resources/schema.sql

```diff
@@ -33,15 +33,19 @@ CREATE TABLE investment (
 
 CREATE TABLE account (
     user_id varchar(255) PRIMARY KEY,
-    amount integer CHECK (amount >= 0)
+    amount integer CHECK (amount >= 0),
+    last_transaction_number integer,
+    last_server varchar(255)
 );
 
 CREATE TABLE account_transaction_log (
-    transaction_num SERIAL PRIMARY KEY,
+    id SERIAL PRIMARY KEY,
     action varchar(255),
     funds integer,
     timestamp bigint,
-    username varchar(255)
+    username varchar(255),
+    server varchar(255),
+    "transactionNum" integer
```

Comment:
how come this has quotes?

Reply:
Postgres is an annoying bitch when it comes to capitalizing column names. The xml tag needs that to be camel case. Xmlforest function turns the column name into the tag name. The profs xml validated needs that tag to be camel case